deploy/consoleplugin/member-operator-console-plugin.yaml

```diff
@@ -3,125 +3,136 @@ kind: Template
-                failureThreshold: 3
-                httpGet:
-                  path: /api/v1/health
-                  port: 8080
-                  scheme: HTTP
-                initialDelaySeconds: 1
-                periodSeconds: 10
-                successThreshold: 1
-                timeoutSeconds: 1
-              readinessProbe:
-                failureThreshold: 30
-                httpGet:
-                  path: /api/v1/health
-                  port: 8080
-                  scheme: HTTP
-                initialDelaySeconds: 1
-                periodSeconds: 1
-                successThreshold: 1
-                timeoutSeconds: 1
-              startupProbe:
-                failureThreshold: 180
-                httpGet:
-                  path: /api/v1/health
-                  port: 8080
-                  scheme: HTTP
-                initialDelaySeconds: 1
-                periodSeconds: 1
-                successThreshold: 1
-                timeoutSeconds: 1
-              env:
-                - name: WATCH_NAMESPACE
-                  value: ${NAMESPACE}
-              resources:
-                requests:
-                  cpu: "50m"
-                  memory: "10M"
-  - kind: Service
-    apiVersion: v1
-    metadata:
-      name: member-operator-console-plugin
-      namespace: ${NAMESPACE}
-      labels:
-        provider: codeready-toolchain
-        run: member-operator-console-plugin
-    spec:
-      ports:
-        - name: "8080"
-          protocol: TCP
-          port: 80
-          targetPort: 8080
-      selector:
-        run: member-operator-console-plugin
-      type: ClusterIP
-      sessionAffinity: null
+          run: member-operator-console-plugin
+      spec:
+        serviceAccountName: member-operator-console-plugin
+        containers:
+        - name: member-operator-console-plugin
+          image: ${IMAGE}
+          ports:
+            - containerPort: 8080
+            - containerPort: 9443
+          command:
+            - member-operator-console-plugin
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /api/v1/status
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 1
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          readinessProbe:
+            failureThreshold: 30
+            httpGet:
+              path: /api/v1/status
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 1
+            periodSeconds: 1
+            successThreshold: 1
+            timeoutSeconds: 1
+          startupProbe:
+            failureThreshold: 180
+            httpGet:
+              path: /api/v1/status
+              port: 8080
+              scheme: HTTP
+            initialDelaySeconds: 1
+            periodSeconds: 1
+            successThreshold: 1
+            timeoutSeconds: 1
```

Comment:
I've noticed that application is exposed on port `9443` (https I guess) , I was wondering if we should have the probes for that port as well ?

Reply:
@sbryzak I wonder if we can just replace the `:8080/api/v1/status` by `:9443/plugin-manifest.json`... Let's just use the manifest as the health check.

Reply:
I just tested it and it works fine. The only problem I see is the logs... Since it's the same endpoint now it's going to be hard to hide the status check pings in the logs.

Reply:
Nice! 👍 

Maybe I've missed what `plugin-manifest.json` is, but indeed testing the same server is a good check!

Reply:
`plugin-manifest.json` is just a file which defines a web-console plugin. This is what the weo-console will do first when loading the plugin. It will download `plugin-manifest.json`. So if it's available then the service is fully ready.

Reply:
Got it, cool! Thanks for the additional details!